site/src/pages/components/selectlist.mdx

```diff
@@ -165,18 +165,40 @@ This example has a `<listbox>` which enables it to put arbitrary content into th
 ```html
 <selectlist>
   <listbox>
-    <div>Heading</div>
-    <div>First two options:</div>
-    <option>one</option>
-    <option>two</option>
-    <div>Second two options:</div>
-    <option>three</option>
-    <option>four</option>
+    <div class=container>
+      <div>
+        <div role=group>1-2</div>
```

Comment:
Is this a "good" example? I would think we should use `<optgroup>` instead of a handful of divs with "role=group", no?

Reply:
I feel like using optgroup here would just be asking for people to say that `<select>` can already handle this use case.
`<optgroup>` has a user agent shadowroot and assigns the actual text with role=group from an attribute, which is not very robust, and `<selectlist>` doesn't have that limitation since youre allowed to put `<div role=group>`s in it.

But yeah this example does not actually do anything that optgroup can't do...

Reply:
I definitely think an example using `<optgroup>` can showcase things the `<select>` can't do. For example, it can contain a fully-styled `<legend>` element (per [the resolution](https://github.com/openui/open-ui/issues/558#issuecomment-1198490800)):

```
<selectlist>
  <optgroup>
    <legend><img src=cool_group_icon>Fancy group</legend>
    <option>...
```